charts/stable/wallabag/values.yaml

```diff
@@ -84,3 +84,20 @@ persistence:
     emptyDir:
       enabled: false
     mountPath: /var/www/wallabag/var/cache
+
+# Enabled postgres
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+postgresql:
+  enabled: false
+  postgresqlUsername: wallabag
+  postgresqlPassword: wallabag-pass
+  postgresqlDatabase: wallabag
+  persistence:
+    enabled: false
+
```

Comment:
Looks like you forgot mariadb in here

Reply:
I copied from home-assistant and it was not there either ;D
I will search the docs and add it for good measure

Reply:
To note: We include just the bare minimum for postgresql anyway...
But yes, it might be a little more clear to at least include mariadb.enabled

Reply:
Well, I actually missed it from home-assistant. Doubled-checked and it is there :shrug: